config/site_settings.yml

```diff
@@ -3532,6 +3532,15 @@ experimental:
     allow_any: false
     refresh: true
     area: "group_permissions|navigation"
+  experimental_admin_search_enabled_groups:
+    hidden: true
+    client: true
+    type: group_list
+    list_type: compact
+    default: ""
+    allow_any: false
+    refresh: true
+    area: "group_permissions"
```

Comment:
Is there an achievement for using all the available properties at once? 😅

Reply:
Assembling Exodia :laughing: